NEWS.md

```diff
@@ -9,6 +9,7 @@
 
 - Templates for ADPC, ADPPK and ADPP are updated to handle urine records. (#2392)
 - `basket_select()` function updated to add `...` argument to allow other qualifiers to be passed to user-defined function specified in `get_terms_fun()` argument for function `create_query_data()`. (#2265)
+- The `id_vars` argument was added to `derive_vars_transposed()` and `derive_vars_atc()` to allow additional variables, beyond those in `by_vars`, to uniquely identify records in `dataset_merge`. (#2325)
```

Comment:
what is dataset_merge?

Reply:
NEWS.md file is updated to specify that `dataset_merge` is an argument.